text/0000-roadmap-2019.md

```diff
@@ -0,0 +1,328 @@
+> - Jonathan Turner, ["The fallow year"](https://www.jonathanturner.org/2018/12/the-fallow-year.html)
+
+> I believe not only that the processes and institutions of Rust need to be
+> shored up, but that special attention should be paid to people. Ultimately,
+> people are what drives the language forward, whether it be through design,
+> implementation, or outreach. If no people want to work on Rust, or are
+> blocked from working on Rust, the language will stagnate. ...
+> 
+> This is not to say that there should not be any movement on improving
+> features or process. Just that their effects and costs should be measured in
+> people, as well as in technology.
+>
+> - Cessen, ["Rust 2019 - It's the Little Things"](https://blog.cessen.com/post/2018_12_12_rust_2019_its_the_little_things)
+
+> In this post, I’ll refer to a highly simplified maturity life cycle with
+> three stages: research, development, and polish. Different parts of the Rust
+> ecosystem are at different levels of maturity. It’s important for effort to
+> match the actual stage in the life cycle, ideally to push it to the next. For
+> example, I consider the language to mostly be in the “polish” stage.
+> Continuing to treat it as research would bring dependent types, virtual
+> structs, etc., which would be interesting but disruptive. Conversely, we
+> don’t know exactly what we want for GUI, so trying to drive that to a
+> standardized solution too early will likely get us to a suboptimal place.
+> 
+> Many mature products have alternating releases focused on new features,
+> then stabilization... 2018 has seen a lot of new features, so I think it’s
+> time for a stabilization phase.
+>  
+> - Raph Levien, ["My thoughts on Rust 2019"](https://raphlinus.github.io/rust/2018/12/16/rust-2019.html)
+
+> If you’re familiar with the history of Rust you might be confused with a
+> call to stabilization. After all, Rust 2015 (a.k.a 1.0) was all about
+> stabilization and the team has actually done a pretty good job with achieving
+> this goal. So what gives?
+> 
+> While Rust 2015 defined stabilization around language backward
+> compatibility, it’s time for the language, the tooling, the ecosystem and the
+> governance to stabilize. Each of these characterize stability in a different
+> way.
+> 
+> - Ryan Levick, ["Rust 2019: Stabilization"](https://blog.ryanlevick.com/posts/rust-2019/)
+
+> The Rust project has been growing like a startup for the last several
+> years. This has some good aspect - “hockeystick curves” - but it has some
+> very bad aspects as well. If this project is going to have a chance of
+> sustaining itself in the long term, we need to get real about dealing with
+> the organizational debt we have accumulated. I think we have serious problems
+> at every level of our organization that need to be addressed, and I’m going
+> to enumerate them from my personal perspective.
+> 
+> - withoutboats, ["Organizational Debt"](https://boats.gitlab.io/blog/post/rust-2019/)
```

Comment:
So while this was one of the posts that I saw a number of quote and at least for me generated a lot of recognition, it feels like the RFC covers very few of the issues, or in general, the topic of organizational debt. Adding back the core team section to suggest work on governance might help with that a little bit, but at least in my view feels like it doesn't give this group of issues the priority that it deserves.